### DISCUSSION TITLE
Depth Map Multiple GPU's

### DISCUSSION THREAD

**Discussion by JonasMoros** [alicevision/Meshroom#2172] [Q&A]
I have two 4090's and I am trying to take advantage of them for processing. I have found that Depth map fails when I set the number of GPU's I have to 2. But when it is one it works just fine. I have CUDA installed on both v12.x and I don't understand why this is happening.

**Comment by JonasMoros** [comment]
I would like to mention if I keep it on two it processes the Depth map right until the last section. Then I fails. Then I have to select use 1 gpu and then it works. When I set it to 0 at the end or 2 it just crashes?

**Reply by natowi** [reply]
Similar issue: https://github.com/alicevision/Meshroom/issues/2166
Are you on windows or linux?
